client/src/routes/Session/components/Slides/Slide.tsx

```diff
@@ -35,7 +37,9 @@ export const Slide = React.memo(({slide, active}: SlideProps) => {
 
   return (
     <Wrapper backgroundColor={theme?.backgroundColor}>
-      {slide.type === 'host' && <Host active={active} />}
+      {slide.type === 'host' && (
+        <Participant participant={participantSpotlight} isHostSpotlight />
+      )}
```

Comment:
I'd like to keep the `Host` slide abstraction but simplify it to this?

Reply:
Interested in the reasoning but let's keep it!